### ISSUE TITLE
`test_build_dataset` is generating named values

### ISSUE THREAD

**Issue by yangsophieee** [traitecoevo/traits.build#30]
Somewhere in process.R, named values are being generated in the final output. This is happening for the `value` column of the `locations` table, and metadata columns (`basis_of_value`, `replicates`, `life_stage`, etc.) in the `traits` and `excluded_data` tables.

**Comment by ehwenk** [comment]
This is creating problems with the tests, because the generated output is named values, while the stored comparison spreadsheet of course just contains values with no names.

Tracing it backward, names are being assigned to each location property values in line 642 of process.R:
```
purrr::map_df(util_list_to_df1, .id = "name") 
```

Which is because of how `value` is created by the function `util_list_to_df1` : 
```
util_list_to_df1 <- function(my_list) {
  for (f in names(my_list)) {
    if (is.null(my_list[[f]]))
      my_list[[f]] <- NA
  }
  tibble::tibble(key = names(my_list), value = unlist(my_list))
}
```
@yangsophieee asks, Is it possible to change `value = unlist(my_list))` to `unname(unlist(my_list))` ? This solves the failing test, but we want to confirm it doesn't create other problems.  @dfalster

**Comment by dfalster** [comment]
Well done identifying the cause. @yangsophieee s suggestion seems good, try it! Alternatively could try 'as.character'. I never intended for the vector to have named elements, so should be fine to remove. Tests will tell us if not!

**Comment by ehwenk** [comment]
closed with [8714247](https://github.com/traitecoevo/traits.build/commit/8714247b22744216c23d2c533f277b576b5d5d5f)
